views.py

```diff
@@ -1,22 +1,23 @@
-from os.path import isfile, join
+from os.path import isfile
 
 from aiohttp import web
 from camilladsp import CamillaError
 
 from filemanagement import (
     path_of_configfile, store_files, list_of_files_in_directory, delete_files,
-    zip_response, zip_of_files, get_yaml_as_json, set_as_active_config, get_active_config, save_config
+    zip_response, zip_of_files, get_yaml_as_json, set_as_active_config, get_active_config, save_config,
+    new_config_with_absolute_filter_paths, coeff_dir_relative_to_config_dir,
+    replace_relative_filter_path_with_absolute_paths, new_config_with_relative_filter_paths
 )
 from offline import cdsp_or_backup_cdsp, set_cdsp_config_or_validate_with_backup_cdsp
 from settings import gui_config_path
 
 try:
-    from camilladsp_plot import plot_pipeline, plot_filter, plot_filterstep
+    from camilladsp_plot import plot_pipeline, eval_filter, eval_filterstep
```

Comment:
questionable if we should keep this. The plotting lib can do all the filter eval without using any extra library (like the big numpy or pyplot) so there shouldn't be anything preventing the backend from requiring it.

Reply:
Cool!
Shall I remove the numpy and pyplot libraries from the README.md?

Reply:
If you don't mind, then yes please go ahead and remove everything about pyplot and numpy!